sigstore-conformance/src/main/java/dev/sigstore/Main.java

```diff
@@ -0,0 +1,125 @@
+package dev.sigstore;
```

Comment:
This needs a copyright header. Preferably with enforcement from spotless (config added above).

`./gradlew spotlessApply` should apply spotless config to all files (and auto add license header)